lvsfunc/deblock.py

```diff
@@ -108,9 +114,14 @@ def _eval_db(n: int, f: List[vs.VideoFrame], clips: List[vs.VideoNode]) -> vs.Vi
     diffnext = core.std.PlaneStats(rgb, rgb.std.DeleteFrames([0]), prop='YNext')
     diffprev = core.std.PlaneStats(rgb, rgb[0] + rgb, prop='YPrev')
 
-    db_clips = [rgb] + [DPIR(rgb, strength=st, task='deblock', device_type='cuda' if cuda else 'cpu',
-                        device_index=device_index).std.SetFrameProp('adb_DeblockStrength', floatval=st) for st in strs]
+    db_clips = [
+        DPIR(rgb, strength=st, task='deblock', device_type='cuda' if cuda else 'cpu',
+             device_index=device_index).std.SetFrameProp('AdbDeblockStrength', floatval=st) for st in strs
```

Comment:
This will currently always set the prop, even if `write_props` is `False`.

Reply:
Indeed. the `write_props` argument is a `more verbose props` like I wrote it in the docstring.

Reply:
I think the param name should be a bit more descriptive itself, then. `more_props` jumps to mind, or something like `verbose`/`quiet`, although that might still be a bit more confusing since we're not printing anything anymore. idk, honestly.